modules/http-helper/http_helper.go

```diff
@@ -28,11 +29,17 @@ func HttpGet(t testing.TestingT, url string, tlsConfig *tls.Config) (int, string
 
 // HttpGetE performs an HTTP GET, with an optional pointer to a custom TLS configuration, on the given URL and
 // return the HTTP status code, body, and any error.
+
+
 func HttpGetE(t testing.TestingT, url string, tlsConfig *tls.Config) (int, string, error) {
 	logger.Logf(t, "Making an HTTP GET call to URL %s", url)
-
+	
 	// Set HTTP client transport config
 	tr := &http.Transport{
+		// Set the proxy from specified environment variables i.e HTTP_PROXY, HTTPS_PROXY, NO_PROXY
+		Proxy: http.ProxyFromEnvironment,
+		// Enable HTTP/2 when TLSClientConfig is provided.
+		ForceAttemptHTTP2: true,
```

Comment:
Would it not make more sense to copy over the defaults from [DefaultTransporter](https://github.com/golang/go/blob/master/src/net/http/transport.go#L42)?

Reply:
> Would it not make more sense to copy over the defaults from [DefaultTransporter](https://github.com/golang/go/blob/master/src/net/http/transport.go#L42)?

Yeah, it actually does.

Reply:
Sorry, what I actually meant was to re-use the values from `DefaultTransporter`, not to copy/paste them. E.g., `Proxy: `DefaultTransporter.Proxy`.

Reply:
> Sorry, what I actually meant was to re-use the values from `DefaultTransporter`, not to copy/paste them. E.g., `Proxy: DefaultTransporter.Proxy`.
I have made the requested changes. Kindly review.